src/index.js

```diff
@@ -60,6 +69,12 @@ class Saturn {
       controller.abort()
     }, options.connectTimeout)
 
+    if (!this.isBrowser) {
+      options.headers = {
```

Comment:
Merge with existing `options.headers` if exists. I'm passing in headers from the arc widget.